panacea/grpc_client.go

```diff
@@ -0,0 +1,99 @@
+package panacea
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/base64"
+	"fmt"
+	"net/url"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/std"
+	oracletypes "github.com/medibloc/panacea-core/v2/x/oracle/types"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+type GRPCClient interface {
+	GetOraclePubKey(ctx context.Context) (*btcec.PublicKey, error)
+}
+
+var _ GRPCClient = &grpcClient{}
+
+type grpcClient struct {
+	conn    *grpc.ClientConn
+	cdc     *codec.ProtoCodec
+	chainID string
```

Comment:
You probably don't need `cdc` and `chainID` for this server.

Reply:
Fixed it on [9e8b9b8](https://github.com/medibloc/panacea-dep-consumer/pull/1/commits/9e8b9b86971082cb64798699e52f753a5551ee6a).